fix test flakiness caused by using sys.getrefcount

walterddr opened the pull request:
Summary: sys.getrefcount() can be flaky before/after scope() call

Test Plan: buck test mode/opt-asan //caffe2/test:others -- 'test_none_names_refcount \(test_namedtensor\.TestNamedTensor\)' --run-disabled

Differential Revision: D24123724

janeyx99 reviewed (approved):
Yay for disabling a flaky test! 🎉  Just wondering though, are there next steps in either fixing it or removing it?

Commit:
fix test flakiness caused by sys.getrefcount(None) (#45876)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/45876

sys.getrefcount() can be flaky before/after scope() call

Test Plan: buck test mode/opt-asan //caffe2/test:others -- 'test_none_names_refcount \(test_namedtensor\.TestNamedTensor\)' --run-disabled

Differential Revision: D24123724

fbshipit-source-id: 277657f3b34d52c72fe4a294ca23fc8e4b43af4d
